code/src/components/app-input-with-icon/AppInputWithIcon.scss

```diff
@@ -0,0 +1,31 @@
+@import "@design-system";
+
+.spa-search-input {
+  padding: spacing(1) spacing(2);
+  display: flex;
+  align-items: flex-end;
+  height: spacing(9);
+  border-radius: 50px;
+  background-color: var(--spa-search-input-background-color, $light);
+
+  &:hover {
+    background-color: var(--spa-search-input-hover-background-color, $white);
```

Comment:
Try to change only variable, not creating a new one

Reply:
Added